server/bootstrap/src/org/labkey/filters/ContentSecurityPolicyFilter.java

```diff
@@ -68,9 +75,16 @@ public void init(FilterConfig filterConfig) throws ServletException
                 policy = s;
                 nonceSubstIndex = policy.indexOf(NONCE_SUBST);
             }
+            else if ("disposition".equals(paramName))
```

Comment:
If we're case insensitive for the values do we want to be for the parameter name too?

Reply:
Hmm, why not.